tests/testthat/test-workflow-bbi.R

```diff
@@ -184,7 +184,7 @@ withr::with_options(list(
     # create model
     mod1 <- read_model(file.path(MODEL_DIR_BBI, "1"))
     submit_model(mod1, .mode = "local", .wait = FALSE)
-    wait_for_nonmem(mod1, 100, interval = 5)
```

Comment:
These tests were failing, but I went ahead and committed the fix in d5f6d9d7af6988c8b172ead4b4db3b3185505afc

We probably missed them because they don't run in CI (because we don't have a NONMEM license on the Drone server).